### PR TITLE
feat: show reactors who reacted to highlights

### PR DESCRIPTION
**Pull request by diivi**
## Description

This PR adds an endpoint that will show who reacted to a highlight and with what reaction.
image

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -4132,13 +4139,13 @@ servers:
   - url: http://localhost:3001
     description: Development
     variables: null
-  - url: https://http:/http:///localhost
+  - url: https://api.opensauced.pizza
```
Comment (bdougie) [inline swagger.yaml:4142]: Does this need to be switched back?

Comment (diivi) [reply swagger.yaml:4142]: Not sure why this got changed 😅, but I'm fine with reverting.

File: src/user/user-highlights.service.ts [hunk 2]
```diff
@@ -273,6 +277,22 @@ export class UserHighlightsService {
     return entities;
   }
 
+  async findAllHighlightReactionsAndReactors(highlightId: number) {
```
Comment (brandonroberts) [inline src/user/user-highlights.service.ts:280]: Why can't this replace the query used in `findAllHighlightReactions`?

Comment (brandonroberts) [reply src/user/user-highlights.service.ts]: As long as it produces the same results for the existing requests, it should be just be adding the array of reactors.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by brandonroberts — APPROVED** [review]
Works as expected locally. Needs merge conflict resolution

**Review by bdougie — APPROVED** [review]

**Pull request merged**